functions.php

```diff
@@ -0,0 +1,41 @@
+<?php
+    /**
+    * Creates a connection to the collectorapp database
+    */
+    function createDBConnection() :PDO{
+        $db = new PDO('mysql:host=db; dbname=collectorapp', 'root', 'password');
+        $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+        return $db;
+    }
+
+    /**
+    * A query to the database that fetches the data in all the fields and creates a recipes assoc array
+    * @return array
+    */
+    function fetchAllRecipes($db): array {
```

Comment:
Check the format of the return type hints, according to psr-12 the format should be like fetchAllRecipes

`function test(): string {`